scripts/falcoctl/falcoctl.yaml

```
@@ -0,0 +1,9 @@
+artifact:
+  follow:
+    every: 6h0m0s
+    falcoversions: http://localhost:8765/versions
```

```suggestion
    falcoVersions: http://localhost:8765/versions
```
Just for legibility ,since viper configuration keys are case insensitive

---

thank you :)